doc/user/content/sql/recursive-ctes.md

```diff
@@ -0,0 +1,297 @@
+---
+title: Recursive CTEs
+description: "Recursive CTEs are used to define recursive queries."
+menu:
+  main:
+    parent: 'transform'
+    name: Recursive CTEs
+    weight: 23
+---
+
+Recursive CTEs can be used to define computations on recursively defined structures (such as trees or graphs) implied by the data in your tables.
```

Comment:
```suggestion
Recursive CTEs explain the recursively-defined structures like trees or graphs implied in your data. Recursive CTEs capture all bindings of the structure and return them in a loop.
```

Reply:
> Recursive CTEs explain the recursively-defined structures like trees or graphs implied in your data.

I would rather say:

> Recursive CTEs _operate on_ the recursively-defined structures like trees or graphs implied in your data.

or

> Recursive CTEs interpret the recursively-defined structures like trees or graphs implied in your data.

and leave the second sentence out.